sentences/fr/light_HassLightSet.yaml

```diff
@@ -2,4 +2,20 @@ language: fr
 intents:
   HassLightSet:
     data:
-      - sentences: []
+      # luminosité
+      - sentences:
+          - "<regle> <name> sur <brightness>"
+          - "<regle> <name> à <brightness>"
+          - "<regle> <lumieres> (dans|de|du) <name> à <brightness>"
+        slots:
+          name: "all"
+      - sentences:
+          - "<regle> <lumieres> de la lampe [<area>] à <brightness>"
+          - "<regle> <lumieres> de la lampe [dans|du|de] [<area>] à <brightness>"
+        slots:
+          name: "light.bedroom_lamp"
```

Comment:
Is it a valid slot ?

Reply:
It's because there is an entity with `lumière de la chambre` as name. We should rename it to another name to avoid confusion.

Reply:
I finally refactored a major part (see my last comment)